packages/docs-reanimated/docs/guides/contributing.mdx

```diff
@@ -0,0 +1,318 @@
+
+If you have spare 2 hours, we highly recommend taking a free [Technical Writing One](https://developers.google.com/tech-writing/one/) course by Google.
+
+##### Avoid passive voice
+
+Most of the sentences in a technical documentation should be in active voice. Sentences in the active voice are usually simpler than the ones in the passive voice.
+
+See the following examples:
+
+- The arguments **are provided to** the function.
+- A worklet **are generated by** a babel plugin.
+- UI thread **is being used by** Reanimated.
+
+which if we rewrite these sentences to active voice become simpler:
+
+- A babel plugin **generates** worklet.
+- The function **receives** the arguments.
+- Reanimated **runs** on the UI thread.
+
+##### Prefer short, clear sentences
+
+When it comes to technical documentation, simplicity is key. Prefer writing short and simple sentences.
+
+Oftentimes you can split long sentences into shorter ones by repeating the sentence subject.
+
+Keep in mind that many readers aren't native English speakers.
+
+##### Split paragraphs into organized lists
+
+People tend to skip over long paragraphs of text. Organized lists are usually easier to read. They help developers get the information they are looking for faster.
+
+Remember to introduce the list first.
+
+```
+The Reanimated installation steps:
+
+- Add Reanimated package form npm
+- Add babel plugin to babel.config.js
+- Reset cache
+```
+
+##### Avoid acronyms and abbreviations
+
+Acronyms can often lead to confusion.
+
+For example, in the context of libraries, abbreviations like `REA`, `RNGH`, `RNS`, may be obvious to developers proficient in the React Native open source space, but completely unknown to beginners.
+
+Some acronyms, such as RN, can mean both React Native and React Navigation.
+
+When explaining topics that often use domain-specific acronyms, such as `JSI`, `JNI`, `JSC`, you should first introduce them. For example, JavaScript Core (JSC) or JavaScript Interface (JSI).
+
+The following rule doesn't apply for a globally established abbreviation like `UI`, `API`, `HTML`, `TS` etc.
+
+#### Use contractions
+
+Prefer writing using contractions like `don't`, `doesn't`, `it's` instead of full forms like `do not`, `does not`, `it is`.
+
+### Embedding interactive examples
+
+We try to make the code examples as easy as it gets. This is why good, interactive examples are crucial. To create interactive examples in the documentation you can use `InteractiveExample` component. It takes `src` prop that expects source of a component that you can declare in `src/examples`, and a `component` prop that showcases given code.
+
+#### Example with a video
+
+```javascript
+import useBaseUrl from '@docusaurus/useBaseUrl';
```

Comment:
is this import important in this example?

Reply:
Fixed in 0b194ed38b115747e70e2cf02f6c5979ac0ae730